miqcli/constants.py

```diff
@@ -63,9 +63,47 @@
     'enable_ssl_verify': False
 }
 
+OS_TYPE = "ManageIQ::Providers::Openstack::CloudManager"
+OS_NETWORK_TYPE = "ManageIQ::Providers::Openstack::NetworkManager"
+
+
+SUPPORTED_AUTOMATE_TYPES = ["gen_floating_ip"]
+
+SUPPORTED_PROVIDERS = ["OpenStack"]
+REQUIRED_OS_KEYS = ["email", "tenant", "image", "security_group", "network",
+                    "flavor", "key_pair", "vm_name"]
+OPTIONAL_OS_KEYS = ["fip_pool"]
+
 #: filesystem root for miqcli authentication store
 AUTHDIR = os.path.join(os.path.expanduser('~'), ".miqcli/auth")
 
+#: time to wait in secs in between checks of a task
+TASK_WAIT = 2
+
+#: time to wait in secs in between checks of a request (provision or automate)
```

Comment:
Same documentation issue here. "between checks of a request" of what?

Reply:
Sorry, this will be taken out, same with the one above.